### PR TITLE
Update the canary writing test

### PR DESCRIPTION
**Pull request by cb372**
It now uses the new Porter canary endpoint to update the lastModified timestamp of a known piece of content and then checks that the timestamp has been updated in Concierge.

We no longer use a collection as the canary, because collections have been deprecated and will soon be removed from CAPI.

Note that due to this change we no longer need to know the basic auth credentials for Porter.

### CODE REVIEW

**Comment by LATaylor-guardian** [comment]
Just a note, we will need to make sure Concierge does not return this 'canary content'. 

Also, we will need to keep the basic auth credentials in Porter to such a time as we can tell them to no longer push to us. (Part of the Shuttlerun clean up work that will be needed)

File: conf/application.conf.sample [hunk 1]
```diff
@@ -14,8 +14,6 @@ content-api-sanity-tests {
     preview-password="capi-preview-password"
     api-key="capi-api-key"
     write-host="https://write-host/"
-    write-username="write-username"
-    write-password="write-password"
```
Comment (gidsg) [inline conf/application.conf.sample]: Also remove these from the Cloudformation template parameters.

**Comment by cb372** [comment]
>Just a note, we will need to make sure Concierge does not return this 'canary content'.

I discussed this with Mariot and we decided we probably don't need to bother filtering it out. It's not associated with any tags, so most people won't find it.

>Also, we will need to keep the basic auth credentials in Porter to such a time as we can tell them to no longer push to us. (Part of the Shuttlerun clean up work that will be needed)

Yep. In my PR comment I just meant that the sanity tests don't need to care about Porter auth anymore.

File: conf/application.conf.sample (hunk 1 above)
Comment (cb372) [inline conf/application.conf.sample]: good point

**Comment by cb372** [comment]
Replaced by #49 and #50

**Pull request closed without merging**